lib/conjur/conjur_config.rb

```diff
@@ -63,5 +71,18 @@ def trusted_proxies_valid?
     rescue
       false
     end
+
+    def authenticators_valid?
+      # TODO: Ideally we would check against the enabled authenticators
+      # in the DB. However, we need to figure out how to use code from the
+      # application without introducing warnings.
+      authenticators_regex = 
+        %r{^(authn|authn-(k8s|oidc|iam|ldap|gcp|azure|config)(/.+)?)$}
```

Comment:
You can remove `config` from here now, right?

Reply:
great catch!